src/Models/Role.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
let's remove role's logic from the package

Reply:
Done.